MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/LineChartActivity1.java

```diff
@@ -126,12 +125,11 @@ protected void onCreate(Bundle savedInstanceState) {
             yAxis.enableGridDashedLine(10f, 10f, 0f);
 
             // axis range
-            yAxis.setAxisMaximum(200f);
-            yAxis.setAxisMinimum(-50f);
+            yAxis.setAxisMaximum(40f);
+            yAxis.setAxisMinimum(0f);
```

Comment:
**Performance Issue**: Changing the axis maximum from '200f' to '40f' and minimum from '-50f' to '0f' might not accurately represent negative values or larger data ranges. <br> **Fix**: Re-evaluate the data range that needs to be represented in the chart. If negative values or values above 40 are expected, adjust the axis minimum and maximum accordingly. <br> **Code Suggestion**: 
 ```
 -            yAxis.setAxisMaximum(200f);
 -            yAxis.setAxisMinimum(-50f);
 +            yAxis.setAxisMaximum(40f);
 +            yAxis.setAxisMinimum(0f);
 ```